components/ai_chat/resources/page/components/header/index.tsx

```diff
@@ -51,7 +49,7 @@ export const PageTitleHeader = React.forwardRef(function (props: FeatureButtonMe
           {isVisible && <Button
             kind='plain-faint'
             fab
-            onClick={() => { aiChatContext.onSelectConversationUuid(undefined) }}
+            onClick={() => location.href = "/"}
```

Comment:
It doesn't seem like we're differentiating anymore between "start a new conversation" and "go to the last conversation for the active tab". Am I wrong about that? I'm doing a build now to verify.

Reply:
New conversation is `/`, once you've started a conversation, the URL will update to include the `uuid`

Reply:
Fixed this!